src/Microsoft.Azure.WebJobs.Host/JobHostConfiguration.cs

```diff
@@ -197,6 +197,18 @@ public INameResolver NameResolver
         }
 
         /// <summary>
+        /// Get the converter manager, which can be used to register additional conversions for 
+        /// customizing model binding. 
+        /// </summary>
+        public IConverterManager ConverterManager
```

Comment:
Add a test for this to the JobHostConfiguration test suite, similar to the StorageClientFactory_GetterSetter test.